doc/build-openbsd.md

```diff
@@ -27,8 +27,8 @@ See [dependencies.md](dependencies.md) for a complete overview.
 **Important**: From OpenBSD 6.2 onwards a C++11-supporting clang compiler is
 part of the base image, and while building it is necessary to make sure that
 this compiler is used and not ancient g++ 4.2.1. This is done by appending
-`CC=cc CC_FOR_BUILD=cc CXX=c++` to configuration commands. Mixing different
-compilers within the same executable will result in errors.
+`CC=cc CXX=c++` to configuration commands. Mixing different compilers within
```

Comment:
23954:
>This is no-longer necessary after upstream libsecp256k1 changes that have been pulled into our repo.

Is this true for our repo?

Reply:
bitcoin#23383 is backported via #5575, it's true for our repo too